src/ip_notifier.cpp

```diff
@@ -339,6 +379,7 @@ struct ip_change_notifier_impl final : ip_change_notifier
 		TORRENT_UNUSED(&ip_change_notifier_impl::on_notify);
 		TORRENT_UNUSED(cb);
 #elif TORRENT_USE_NETLINK
+		using namespace std::placeholders;
```

Comment:
all mentions of ``TORRENT_USE_NETLINK`` and associated snippets of code can be removed now, right?
this function and the constructor specifically